messages/th.json

```diff
@@ -568,5 +568,7 @@
   "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "URL มาตรฐานที่ไม่มี URL Params",
   "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "ลบพารามิเตอร์  Map ออกจาก URL มาตรฐาน อาจเกิดปัญหาเมื่อไปยังหน้าเพจด้วยแฟเซ็ตที่เลือก โปรดทดสอบด้วยความระมัดระวังหลังเปิดใช้",
   "admin/store.advancedSettings.removeStoreNameTitle.title": "ลบชื่อร้านค้าออกจากคำเรียก",
-  "admin/store.advancedSettings.removeStoreNameTitle.description": "ลบชื่อร้านค้าออกจากตอนท้ายของชื่อผลิตภัณฑ์และชื่อ Meta -ของหมวดหมู่"
+  "admin/store.advancedSettings.removeStoreNameTitle.description": "ลบชื่อร้านค้าออกจากตอนท้ายของชื่อผลิตภัณฑ์และชื่อ Meta -ของหมวดหมู่",
+  "admin/actions.enable-binding": "เปิดใช้ค่ากำหนดด้วยการผูกค่า",
+  "admin/actions.binding-configurations": "Binding configurations"
```

Comment:
Is it supposed to be in English?

Reply:
Yes, we still have to request the translation of this one, and because it's so short I'm collecting more strings to send along.